formats/qcow.nix

```diff
@@ -1,4 +1,4 @@
-{ config, lib, pkgs, modulesPath, ... }:
+{ config, lib, pkgs, modulesPath, diskSize ? 8192, ... }:
```

Comment:
how would you override that?

Reply:
I would say it's probably better to have something like nixos module option here? Otherwise it's only overridable by using some obscure less known interfaces like:

```
 {                                               
   _module.disk.diskSize = 8192;                     
 }
```